.github/workflows/docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Markdown Linter
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+  pull_request:
+    paths-ignore:
+      - './**/*.md'
```

Comment:
This workflow will ignore all PRs with `*.md` changes. Seems like it is a mistake.